src/ba.py

```diff
@@ -1,7 +1,23 @@
 """Module for computing border arrays."""
+import numpy as np
 
+def border_array(x):
 
-def border_array(x: str) -> list[int]:
+    if x == '':
+        return []
+
+    ba = np.zeros(len(x), dtype = int)
+    ba[0] = 0
```

Comment:
This isn't necessary if you already initialised `ba` to all zeros.

Reply:
Nåh ja. Mange tak